packages/verbum-block-editor/.gitignore

```diff
@@ -0,0 +1 @@
+.cache
```

Comment:
Let's add a `/packages/*/.cache` record to the top-level `.gitignore` instead. There's already an identical one for `apps/...` there.

Reply:
Applied the suggestion 👍